Example/Ohana/OHBasicContactPhotosPicker.swift

```diff
@@ -81,29 +81,29 @@ class OHBasicContactPhotosPicker : UITableViewController, OHCNContactsDataProvid
 
     // MARK: UITableViewDataSource
 
-    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+    override func numberOfSections(in tableView: UITableView) -> Int {
         return 1
     }
 
-    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         if let contacts = dataSource.contacts {
             return contacts.count
         }
         return 1
     }
 
-    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        let cell = UITableViewCell(style: .Default, reuseIdentifier: nil)
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = UITableViewCell(style: .default, reuseIdentifier: nil)
 
-        if let contact = self.dataSource.contacts?.objectAtIndex(indexPath.row) as? OHContact {
+        if let contact = self.dataSource.contacts?.object(at: (indexPath as NSIndexPath).row) as? OHContact {
```

Comment:
Can we not just do `indexPath.row`?  Looks like Swift's `IndexPath` has a `row` property.